app/src/main/res/values/text_styles.xml

```diff
@@ -9,6 +9,10 @@
         <item name="fontPath">@string/AvenirNext_DemiBold</item>
     </style>
 
+    <style name="AvenirNext_Bold" parent="android:TextAppearance">
+        <item name="fontPath">@string/AvenirNext_Bold</item>
```

Comment:
I see it in the fonts directory on my end. https://github.com/IntrepidPursuits/ContestAndroid/blob/abass/con26scoreAnEntry/app/src/main/assets/fonts/AvenirNext-DemiBold.ttf